### PR TITLE
add ansible collection details.

### PR DESCRIPTION
**Pull request by rayashworth**

### CODE REVIEW

File: collection.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+```
+
+## Using in your playbook
+
+```
+---
+- name: Prepare cluster for Demo Deployment
+  hosts: localhost
+  gather_facts: no
+  tasks:
```
Comment (websterc87) [inline collection.md:37]: Should we switch this to roles, rather than encouraging include_role: too much.

Comment (rayashworth) [reply collection.md:37]: I passed this question on to Rahul

Comment (ratripathi) [reply collection.md:37]: Sure we can do that as well. The reason I showed that `include_role` was to showcase on how a task can also be included.

**Review by wkrapohl — APPROVED** [review]
Good

File: collection.md (hunk 1 above)
Comment (rayashworth) [reply collection.md:49]: added "roles:" example

**Review by ratripathi — APPROVED** [review]
LGTM, thanks Ray!

**Pull request merged**